modulefiles/srw_common.lua

```diff
@@ -1,26 +1,26 @@
 load_any("jasper/2.0.25","jasper/2.0.32")
 load_any("zlib/1.2.11","zlib/1.2.13")
-load("libpng/1.6.37")
+load_any("png/1.6.37","libpng/1.6.37")
 
-load_any("netcdf/4.9.2", "netcdf-c/4.9.2")
-load_any("netcdf/4.9.2", "netcdf-fortran/4.6.0")
-load_any("pio/2.5.10", "parallelio/2.5.9")
+load_any("netcdf/4.9.2","netcdf-c/4.9.2")
+load_any("netcdf/4.9.2","netcdf-fortran/4.6.0")
+load_any("pio/2.5.10","parallelio/2.5.9")
```

Comment:
When Hercules spack-stack is included, we'd need parallelio/2.5.10, please change line 7 to the following:
load_any("pio/2.5.10","parallelio/2.5.9","parallelio/2.5.10")